disnake/flags.py

```diff
@@ -829,6 +829,7 @@ class Intents(BaseFlags):
 
     To construct an object you can pass keyword arguments denoting the flags
     to enable or disable.
+    Arguments are applied in order, similar to :class:`Permissions`.
```

Comment:
this should also be under the other times that 
> `To construct an object you can pass keyword arguments denoting the flags` 

is mentioned

Reply:
Those other types currently don't have any aliased attributes which is why I didn't add it initially, but fine by me
https://github.com/DisnakeDev/disnake/pull/749/commits/9d3c82903aedbe5b4226a8c6259ab1c57e0d609b